ckanext/xloader/jobs.py

```diff
@@ -128,19 +117,25 @@ def xloader_data_into_datastore_(input, job_dict):
     # also show logs on stderr
     logger.addHandler(logging.StreamHandler())
     logger.setLevel(logging.DEBUG)
+    # also log to file
+    if config.get('ckanext.xloader.jobs_log'):
+        file_handler = logging.FileHandler(config.get('ckanext.xloader.jobs_log'))
+        file_handler.setLevel(logging.DEBUG)
+        file_handler.setFormatter(logging.Formatter('%(asctime)s %(levelname)-5.5s [%(name)s] %(message)s'))
+        logger.addHandler(file_handler)
```

Comment:
logging handlers should be configured from the ini not hard-coded

Reply:
@wardi okay the logging if better now. The database logging handler has to stay in of course. But the standard Stream logging and the new File logging is handled in the INI now. PYTHON!